cypress/support/pages/dashboard/pageDashboard.js

```diff
@@ -1,32 +1,29 @@
 'use strict'
 
 class Dashboard {
+
   static #welcomeMessage = '[data-testid="welcome-section"]'
   static #dashboardTasksTitle = '[data-testid="dashboard-tasks-title"]'
   static #dashboardTaskCourseName = '[data-testid="task-course-name"]'
   static #dashboardTaskName = '[data-testid="task-name"]'
 
-  seeWelcomeMessage () {
+  seeWelcomeMessage() {
     cy.get(Dashboard.#welcomeMessage)
     cy.contains('Hallo')
   }
 
-  seeAssignedTasks () {
+  seeAssignedTasks() {
     cy.get(Dashboard.#dashboardTasksTitle).eq(0)
     cy.contains('Gestellte Aufgaben')
```

Comment:
This depends on the choosed language, right?
We should consider that we change it to be independent from the chosen language.
BTW: We have an attribute lang='de'/'en'/'es'/'ua' in the html-tag to see which language is chosen.

Reply:
yes, created a follow up ticket (#2039) since it was already on the main before.